examples/measurements/README.md

```diff
@@ -0,0 +1,16 @@
+# Measurements
+
+This folder contains several programs to measure the energy and time it takes for the different algorithms to complete.
+
+## Compiling for ATmega
+
+The measurement programs are compiled automatically, when compiling the `libmicrofido2`.
+
+## Compiling for nRF52
+
+The examples must be compiled explicitly for the nRF52.
+
+First, navigate to the [nrf52](./nrf52) folder.
+Then, make sure to select the algorithm you want to measure in the `CMakeLists.txt` by modifying the `set(MEASURE_ALGORITHM aes_gcm)`.
+Finally, the build procedure is similar to the one in the [nrf52 example](../nrf52/README.md).
+To build and flash the program, execute: `docker run --rm -v $PWD/../../../:/libmicrofido2 --privileged --device=/dev/ttyACM? nrf52-sdk west flash -d /libmicrofido2/examples/measurements/nrf52/build`.
```

Comment:
Doesn't this only flash and not build?

Reply:
The `flash` command first checks whether a rebuild is necessary and if so performs a fresh build. Then it flashes the image.

Reply:
I get `FATAL ERROR: no CMake cache found (expected one at /libmicrofido2/examples/measurements/nrf52/build/CMakeCache.txt)`

Reply:
ah ok

Reply:
Running:
```
cmake . -DCMAKE_VERBOSE_MAKEFILE=1 -DCMAKE_BUILD_TYPE=Debug
```
returns
```
CMake Error at CMakeLists.txt:8 (find_package):
  Could not find a package configuration file provided by "Zephyr" with any
  of the following names:

    ZephyrConfig.cmake
    zephyr-config.cmake

  Add the installation prefix of "Zephyr" to CMAKE_PREFIX_PATH or set
  "Zephyr_DIR" to a directory containing one of the above files.  If "Zephyr"
  provides a separate development package or SDK, be sure it has been
  installed.


-- Configuring incomplete, errors occurred!
```

Reply:
I assume you run that command outside of any Docker container. If you haven't installed the Zephyr framework locally, building won't work.
Moreover, I'm unsure of the parameters and other magic that the west command passes on to cmake, so I would always use `West build` rather than `cmake` directly.

Reply:
The cmake thing was just an attempt to fix the previous error

Reply:
Ah, I think the problem is that running `flash` before having done a `build` at least once (so the directory contains a cache which flash can check) won't work. So you'll need the `west build` command first.

Reply:
So should I add that to the README?